footer.js

```diff
@@ -33,7 +33,8 @@ document.write('\
                 </div>\
             </div>\
             <div class="ui inverted section divider"></div>\
-            <img src="./assets/images/kosslogo.png" class="ui centered mini image">\
+            <img src="../assets/images/kosslogo.png" class="ui centered mini image" alt="">\
+            <img src="./assets/images/kosslogo.png" class="ui centered mini image" alt="">\
```

Comment:
I can't understand why but it is appearing only once on my laptop.
![screenshot from 2017-05-09 10-39-41](https://cloud.githubusercontent.com/assets/24224542/25836383/d4a6fac4-3475-11e7-8941-5ee4ee04f7f5.png)